lms/djangoapps/discussion/plugins.py

```diff
@@ -1,33 +1,25 @@
 """
 Views handling read (GET) requests for the Discussion tab and inline discussions.
 """
+from django.utils.translation import ugettext_lazy as _
 
+from openedx.core.djangoapps.discussions.discussions_apps import DiscussionApp
+from .django_comment_client.utils import is_discussion_enabled
 
-from django.conf import settings
-from django.utils.translation import ugettext_noop
 
-import lms.djangoapps.discussion.django_comment_client.utils as utils
-from lms.djangoapps.courseware.tabs import EnrolledTab
-from xmodule.tabs import TabFragmentViewMixin
-
-
-class DiscussionTab(TabFragmentViewMixin, EnrolledTab):
+class CommentServiceDiscussionApp(DiscussionApp):
     """
-    A tab for the cs_comments_service forums.
+    Discussion Plugin app for cs_comments_service.
     """
+    name = "cs_comments"
+    friendly_name = _("Inbuilt Discussion Forums")
```

Comment:
Nit: @marcotuts - I feel like we may want a different friendly name than this.  Is this "Default Forums" or "edX Forums" or something?

Reply:
thanks for calling this out - "edX Discussions" is good for now.

Reply:
Done.